### PR TITLE
Samples: Bluetooth: ISO time sync sample on 54h

### PR DESCRIPTION
**Pull request by Tschet1**
Port the ISO time sync sample to 54h. This works out of the box and only some additions are needed to the device tree configuration  set up the DPPI connection between the GRTC and the LED.

### CODE REVIEW

**Review by rugeGerritsen — APPROVED** [review]

**Review by yuxCai — APPROVED** [review]

File: samples/bluetooth/iso_time_sync/README.rst [hunk 1]
```diff
@@ -52,9 +52,11 @@ The sample demonstrates the following features:
...
 
 .. note::
-   This sample requires less hardware resources when it is run on an nRF54L Series device compared to the nRF52 or nRF53 Series devices.
-   On an nRF54L Series device, only one GRTC channel and PPI channel is needed to set up accurate toggling of an LED.
-   On nRF52 and nRF53 Series devices, you also need one RTC peripheral, one TIMER peripheral, one EGU channel, three PPI channels, and one PPI group.
+   This sample requires less hardware resources when it is run on an nRF54 Series device compared to the nRF52 or nRF53 Series devices.
+   On an nRF54L Series device, only one GRTC channel and DPPI channel is needed to set up accurate toggling of an LED.
+   On an nrf54H Series device, only one GRTC channel and two local DPPI channels are needed to connect the GRTC and the LED to the local PPIB instances.
+   Additionally, a PPIB channel is used to connect the two PPIB instances.
+   On nRF52 and nRF53 Series devices, you need one RTC peripheral, one TIMER peripheral, one EGU channel, four PPI channels, and one PPI group.
```
Comment (divipillai) [inline samples/bluetooth/iso_time_sync/README.rst:59]:
```suggestion
   This sample requires less hardware resources when it is run on an nRF54 Series device compared to the nRF52 or nRF53 Series devices.
   
   * On an nRF54L Series device, only one GRTC channel and DPPI channel is needed to set up accurate toggling of an LED.
   * On an nrf54H Series device, only one GRTC channel and two local DPPI channels are needed to connect the GRTC and the LED to the local PPIB instances.
   Additionally, a PPIB channel is used to connect the two PPIB instances.
   * On nRF52 and nRF53 Series devices, you need one RTC peripheral, one TIMER peripheral, one EGU channel, four PPI channels, and one PPI group.
```
For better readability.

File: samples/bluetooth/iso_time_sync/README.rst [hunk 2]
```diff
@@ -52,9 +52,12 @@ The sample demonstrates the following features:
       This allows you to measure the minimal end-to-end latency.
 
 .. note::
-   This sample requires less hardware resources when it is run on an nRF54L Series device compared to the nRF52 or nRF53 Series devices.
-   On an nRF54L Series device, only one GRTC channel and PPI channel is needed to set up accurate toggling of an LED.
-   On nRF52 and nRF53 Series devices, you also need one RTC peripheral, one TIMER peripheral, one EGU channel, three PPI channels, and one PPI group.
+   This sample requires less hardware resources when it is run on an nRF54 Series device compared to the nRF52 or nRF53 Series devices.
```
Comment (richabp) [inline samples/bluetooth/iso_time_sync/README.rst:55]:
```suggestion
   This sample requires fewer hardware resources when it is run on an nRF54 Series device compared to the nRF52 or nRF53 Series devices.
```

File: samples/bluetooth/iso_time_sync/README.rst [hunk 3]
```diff
@@ -52,9 +52,12 @@ The sample demonstrates the following features:
...
+
+   * On an nRF54L Series device, only one GRTC channel and DPPI channel is needed to set up accurate toggling of an LED.
```
Comment (richabp) [inline samples/bluetooth/iso_time_sync/README.rst:57]:
```suggestion
   * On an nRF54L Series device, only one GRTC channel and one DPPI channel are needed to set up accurate toggling of an LED.
```

File: samples/bluetooth/iso_time_sync/README.rst [hunk 4]
```diff
@@ -52,9 +52,12 @@ The sample demonstrates the following features:
...
+   * On an nrf54H Series device, only one GRTC channel and two local DPPI channels are needed to connect the GRTC and the LED to the local PPIB instances.
```
Comment (richabp) [inline samples/bluetooth/iso_time_sync/README.rst:58]:
```suggestion
   * On an nRF54H Series device, only one GRTC channel and two local DPPI channels are needed to connect the GRTC and the LED to the local PPIB instances.
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by divipillai — APPROVED** [review]

**Review by richabp — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

**Pull request merged**